src/Combinatorics/EnumerativeCombinatorics/combinations.jl

```diff
@@ -110,16 +116,16 @@ function Base.getindex(C::Combination, i::IntegerUnion)
   return getindex(data(C), Int(i))
 end
 
-function Base.copy(C::Combination)
-  return Combination(copy(data(C)))
+function Base.copy(C::Combination{T}) where T
+  return Combination{T}(copy(data(C)))
 end
 
 function Base.getindex(C::Combinations{Base.OneTo}, i::IntegerUnion)
-  return Oscar.combination(C.n, C.k, i)
+  return combination(C.n, C.k, i)
```

Comment:
BTW I don't think this will work if `i` is anything but an `Int` given that `C.n` and `C.k` are currently always `Int` (perhaps we should change that, though?)